src/Features/Core/Portable/ExtractMethod/MethodExtractor.CodeGenerator.cs

```diff
@@ -56,6 +57,7 @@ protected CodeGenerator(InsertionPoint insertionPoint, SelectionResult selection
             protected abstract Task<SyntaxNode> GenerateBodyForCallSiteContainerAsync(CancellationToken cancellationToken);
             protected abstract SyntaxNode GetPreviousMember(SemanticDocument document);
             protected abstract OperationStatus<IMethodSymbol> GenerateMethodDefinition(bool localFunction, CancellationToken cancellationToken);
+            protected abstract Task<(ExpressionBodyPreference expressionBodiedMethod, ExpressionBodyPreference expressionBodiedLocalFunction, bool staticLocalFunction)> StaticLocalFunctionAndExpressionBodyPreferenceAsync(SemanticDocument semanticDocument, CancellationToken cancellationToken);
```

Comment:
general rule of thumb: if it scrolls this long on github, plz wrap.  the experience is just painful to revew otherwise :) 

It's ok if it goes a teensy bit past the github margin... but this is very excessive :)